### DISCUSSION TITLE
Can't apply css & JS on GitHub Pages

### DISCUSSION THREAD

**Discussion by OverFishy** [community/community#23102] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>Iv’e tried to deploy my sudoku project on GitHub Pages platform, HTML seems to go by, but JS &amp; CSS are not being applied.</p>
<p>I’ve tried some solutions, and feel kind of stuck.</p>
<p>Github repo: <a href="https://github.com/OverFishy/soduko-mobile" class="inline-onebox" rel="noopener nofollow ugc">GitHub - OverFishy/soduko-mobile</a><br>
Github pages: <a href="https://overfishy.github.io/soduko-mobile/" class="inline-onebox" rel="noopener nofollow ugc">soduko-mobile</a></p>
<p>Thank you in advance fo any pointers.</p>
<p>Ofek.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>it’s loading the README,</p>
<p><a href="https://overfishy.github.io/soduko-mobile/">https://overfishy.github.io/soduko-mobile/</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/3/d39e58c236833d654cf314ba04aef598829d7f6e.png" data-download-href="/uploads/short-url/uc409JQHcauCSUfQBZyjCfs74rI.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074864-9b92d25c-38e1-499e-a961-c3190237616f.png" alt="image" data-base62-sha1="uc409JQHcauCSUfQBZyjCfs74rI" width="690" height="407" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d39e58c236833d654cf314ba04aef598829d7f6e_2_690x407.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d39e58c236833d654cf314ba04aef598829d7f6e_2_1035x610.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/3/d39e58c236833d654cf314ba04aef598829d7f6e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d39e58c236833d654cf314ba04aef598829d7f6e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1140×673 77.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>but if you type the exact URL, you’ll get</p>
<p><a href="https://overfishy.github.io/soduko-mobile/sudoku.html">https://overfishy.github.io/soduko-mobile/sudoku.html</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/5/85f5d8ff29b2dc7dbfae04428f911fc8853a3e92.png" data-download-href="/uploads/short-url/j74ef1UdXZiYUDsRZlCVhyNPzJE.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074865-c47df943-3da3-4d3b-8115-88ccd33d874f.png" alt="image" data-base62-sha1="j74ef1UdXZiYUDsRZlCVhyNPzJE" width="690" height="407" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85f5d8ff29b2dc7dbfae04428f911fc8853a3e92_2_690x407.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85f5d8ff29b2dc7dbfae04428f911fc8853a3e92_2_1035x610.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/5/85f5d8ff29b2dc7dbfae04428f911fc8853a3e92.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85f5d8ff29b2dc7dbfae04428f911fc8853a3e92_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1140×673 85.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
